typescript/processors/readTypeScriptModules.js

```diff
@@ -0,0 +1,451 @@
+var glob = require('glob');
+var path = require('canonical-path');
+var _ = require('lodash');
+var ts = require('typescript');
+
+module.exports = function readTypeScriptModules(tsParser, modules, getFileInfo, untouchedNamespaces,
+                                                getExportDocType, getContent, createDocMessage, log) {
+
```

Comment:
Hey @pvadam: might be due to the globbing pattern. I will have a look on the weekend ...